git-cliff-core/src/changelog.rs

```diff
@@ -144,6 +147,26 @@ impl<'a> Changelog<'a> {
 				})
 				.collect::<Vec<Commit>>();
 		});
+
+		if self.config.git.require_conventional.unwrap_or(false) {
+			debug!("Verifying that all commits are conventional.");
+			let mut unconventional_count = 0;
+
+			self.releases.iter().for_each(|release| {
+				release.commits.iter().for_each(|commit| {
+					if commit.conv.is_none() {
+						error!("Commit {} is not conventional.", &commit.id[..7]);
```

Comment:
How about we also log the commit message here?

```suggestion
						error!(
							"Commit {id} is not conventional. ({message})",
							id = &commit.id[..7],
							message = commit
								.message
								.lines()
								.next()
								.unwrap_or_default()
								.trim()
						);
```

Reply:
I was considering it, but I was unsure how this ends up looking in the log. Given the following commit message, only the first line ends up being logged:

```txt
This is unconventional on multiple lines.

Line 2

Some Footer: user3 <user3@example.com>
```

```txt
Commit ffdc50e is not conventional. (This is unconventional on multiple lines.)
```

Would it make sense to log the message on separate lines underneath instead?

Reply:
Having multi-line error messages is bad for automated processing of log messages, but I doubt that would be a problem for git-cliff, because it is a pure CI tool.

Reply:
Yup, then we can do format it like that when there is a multi line commit and print everything in a single line when it's a single line commit.

Alternatively I was thinking to add `colored` to make it prettier (e.g. make the commit ID orange etc,), but I'm not sure.

Reply:
I changedit to the new format for both single- and multi-line commits. I think this is fine and easyon the eyes.

```txt
ERROR git_cliff_core::changelog > Commit df5e6aa is not conventional:
    | This is unconventional on a single line.
ERROR git_cliff_core::changelog > Commit ffdc50e is not conventional:
    | This is unconventional on multiple lines.
    |
    | Line 2
    |
    | Some Footer: user3 <user3@example.com>
```